File: televisionbingecalculator/src/main/java/com/vanillax/televisionbingecalculator/app/ServerAPI/TV/CastResponse.java
package com.vanillax.televisionbingecalculator.app.ServerAPI.TV;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by mitchross on 5/21/17.
 */

public class CastResponse
{
	@SerializedName("id" )
	public int id;

	@SerializedName( "cast" )
	public List<Cast> castList;
}

File: televisionbingecalculator/src/main/java/com/vanillax/televisionbingecalculator/app/serverapi/TV/CastResponse.kt
package com.vanillax.televisionbingecalculator.app.serverapi.TV

import com.google.gson.annotations.SerializedName
import com.vanillax.televisionbingecalculator.app.kotlin.network.response.Cast

/**
 * Created by mitchross on 5/21/17.
 */

class CastResponse {
    @SerializedName("id")
    var id: Int = 0

    @SerializedName("cast")
    var castList: List<Cast>? = null
}
